### ISSUE TITLE
Record date time of when analysis was run

### ISSUE THREAD

**Issue by sarahLy9** [poissonconsulting/embr#76]
Would help keep track of when models were last ran and if any changes had been made to the data after that point requiring a rerun of the model.

**Comment by joethorley** [comment]
@sarahLy9 I'm thinking this information is best got from the age of the object in the subfoldr2 directory? @aylapear what do you think?

**Comment by joethorley** [comment]
In this way such a function could be used for other objects or types.

**Comment by sarahLy9** [comment]
Is there any potential for this to be effected by copying files and therefore artificially bumping up the creation date? I know it's rare for the output folder to be copied but we have done it before in certain circumstances.

**Comment by joethorley** [comment]
Yes you're totally right. A general solution would be to add a text file that logs the recent version (and updates?). Could be useful as a check that everything is up to date?

**Comment by aylapear** [comment]
I'd say generally yes a person could use the last modified date of the objects in the output folder to determine when things were run last but it would take a bit of cross checking if you had to check the date of the data, commit history and model output. If files were copied or moved around it is highly likely these dates would not be representative of when the model was run. 

We could generate a text file indicating some details of the run. I would suggest making those changes in our template files then in embr itself.

**Comment by joethorley** [comment]
Yes I agree - I think this should be move to subfoldr2 as it would be handled by its infrastructure.

**Comment by sarahLy9** [comment]
I created a new issue for this in subfolder2 repo.
